components/Navbar.tsx

```
@@ -0,0 +1,307 @@
+  Phone,
+  Target,
+  FileSearch,
+  TrendingUp,
+  BarChart3,
+  PieChart,
+  ChevronDown,
+  ChevronRight,
+  Video,
+  MessageCircle,
+  CalendarDays,
+  Map,
```

Biome (2.1.2)

[error] Do not shadow the global "Map" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)